examples/rfid_read_personal_data/rfid_read_personal_data.ino

```diff
@@ -0,0 +1,141 @@
+/*
+ * Reads data written by a program such as "rfid_write_personal_data.ino" 
+ * 
+ * See: https://github.com/miguelbalboa/rfid/tree/master/examples/rfid_write_personal_data
+ * 
+ * Uses MIFARE RFID card using RFID-RC522 reader
+ * Uses MFRC522 - Library  
+ * -----------------------------------------------------------------------------------------
+ *             MFRC522      Arduino       Arduino   Arduino    Arduino          Arduino
+ *             Reader/PCD   Uno/101       Mega      Nano v3    Leonardo/Micro   Pro Micro
+ * Signal      Pin          Pin           Pin       Pin        Pin              Pin
+ * -----------------------------------------------------------------------------------------
+ * RST/Reset   RST          9             5         D9         RESET/ICSP-5     RST
+ * SPI SS      SDA(SS)      10            53        D10        10               10
+ * SPI MOSI    MOSI         11 / ICSP-4   51        D11        ICSP-4           16
+ * SPI MISO    MISO         12 / ICSP-1   50        D12        ICSP-1           14
+ * SPI SCK     SCK          13 / ICSP-3   52        D13        ICSP-3           15
+ *
+ */
+ 
+#include <SPI.h>
+#include <MFRC522.h>
+
+#define RST_PIN         9           //see pin layout
+#define SS_PIN          10          //see pin layout
+
+MFRC522 card(SS_PIN, RST_PIN);   // Create MFRC522 object 'card'
```

Comment:
please rename `card` to `mfrc522` , because all other examples use the same wording